### ISSUE TITLE
Add more maintenance options to safer-cluster module

### ISSUE THREAD

**Issue by christiangonre** [terraform-google-modules/terraform-google-kubernetes-engine#1281]
### TL;DR

The [beta private cluster module](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/v21.1.0/modules/beta-private-cluster) used from [safer-cluster module](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/4a7e6e40e923a37d922e58027f2bfb0a43bb32af/modules/safer-cluster/main.tf#L22) has an option to enable **maintenance_end_time**, **maintenance_exclusions** and **maintenance_recurrence**.
It would be awesome if we could specify it from the safer-cluster module too.

### Detailed design

```markdown
I will copy those variables from the module beta-private-cluster and enable it on the safer-cluster module
```

### Additional information

I will commit a pull request with the solution. Thanks for these modules, they are fantastic!
